tests/integration/weight_test.go

```diff
@@ -0,0 +1,37 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/rancher/rio/tests/testutil"
```

Comment:
Hmm, so should we do imports similar to rancher? 
std lib imports are grouped and all others are grouped in another block? 
I think we should

Reply:
Switched to the 3 groups: stdlib, external, and internal